Code/Database/SQLCreateQuery.sql

```diff
@@ -1,6 +1,8 @@
 USE master
-
-DROP DATABASE GTL;
+go
+IF DB_ID('GTL') IS NOT NULL
+ALTER DATABASE GTL SET SINGLE_USER WITH ROLLBACK IMMEDIATE
+DROP DATABASE IF EXISTS GTL;
```

Comment:
what are the things being added here?

Reply:
to make sure that even if the db don't exist in the first place, sqlcmd can still run the script fully.
the Alter db part is because sometimes ssms or some other process randomly blocks sqlcmd access to eventually existing gtl db.